virtcontainers/default_network.go

```diff
@@ -54,6 +55,10 @@ func (n *defNetwork) add(s *Sandbox) error {
 		return err
 	}
 
+	sort.Slice(endpoints, func(i, j int) bool {
```

Comment:
Good catch! But why not put this logic into `createEndpointsFromScan()` itself (and document that the returned slice is sorted by endpoint name in a comment above the function)?

Reply:
In fact, please could you add a log call (ideally in `createEndpointsFromScan()` where the sorting could happen) showing the eventual order of endpoints. That will make debugging any such issues in future a little easier :)